core-bundle/contao/languages/en/default.xlf

```diff
@@ -2270,6 +2270,18 @@
       <trans-unit id="MSC.removeItem">
         <source>Remove item</source>
       </trans-unit>
+      <trans-unit id="MSC.ariaMenuLabel">
+        <source>Menu</source>
+      </trans-unit>
+      <trans-unit id="MSC.ariaMenuToggle">
+        <source>Toggle menu</source>
```

Comment:
It would be more helpful if the button had _open menu_  and _close menu_ labels instead of a generic _toggle menu_ that does not say anything about the current status. But maybe the `aria-expanded` attribute is enough here?

Reply:
We could also use the ariaLabel as a title on the button so it would indicate the menu for blind users.

Reply:
I decided to use the configured `aria-label` so screenreaders would read out the button. The `aria-expanded` attributes are also correctly telling you about the state if it's open or not.

Basically our case [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label#description):
> Sometimes, the default <a href="/en-US/docs/Glossary/Accessible_name">accessible name</a> of an element is missing or the accessible name does not accurately describe the contents of the element and there is no content visible in the DOM that can be associated with the object to give it meaning. A common example of such an element is a button containing an SVG icon without any text.


See: 3b1f7306412f806a17fc10ed0eba3221cc812c97